dali/kernels/imgproc/jpeg/jpeg_distortion_gpu_kernel.h

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_KERNELS_IMGPROC_JPEG_JPEG_DISTORTION_GPU_KERNEL_H_
+#define DALI_KERNELS_IMGPROC_JPEG_JPEG_DISTORTION_GPU_KERNEL_H_
+
+#include <vector>
+#include "dali/core/convert.h"
+#include "dali/core/geom/mat.h"
+#include "dali/core/geom/vec.h"
+#include "dali/core/tensor_view.h"
+#include "dali/core/util.h"
+#include "dali/kernels/common/block_setup.h"
+#include "dali/kernels/common/utils.h"
+#include "dali/kernels/kernel.h"
+
+namespace dali {
+namespace kernels {
+namespace jpeg {
+
+inline float GetQualityFactorScale(int quality) {
+  quality = clamp<int>(quality, 1, 100);
+  float q_scale = 1.0f;
+  if (quality < 50) {
+    q_scale = 50.0f / quality;
+  } else {
+    q_scale = 2.0f - (2 * quality / 100.0f);
```

Comment:
```suggestion
    q_scale = 2.0f - (2 * quality * .001f);
```

Reply:
this is only run once, so I prefer to keep the division, as it reads better.